CMakeLists.txt

```diff
@@ -50,7 +50,12 @@ IF(MQTT_SUPPORT)
         FIND_LIBRARY(PAHO_MQTT_LIB NAMES paho-mqtt3a paho-mqtt3c paho-mqtt3as paho-mqtt3cs)
 
         IF(NOT PAHO_MQTT_LIB)
-            MESSAGE(FATAL_ERROR "Cannot find Paho MQTT library.")
+            FIND_LIBRARY(MOSQUITTO_LIB NAMES mosquitto)
```

Comment:
Discussion later: mqtt lib priority (1. mosquitto 2. paho)

Reply:
After implementing mosquitto wrapper, lets determine the priority of mqtt libs.

Reply:
Checked. Let's determine the priority after implementing with mosquitto lib.